### PR TITLE
Upgrade/swift5

### PR DESCRIPTION
**Pull request by HarshalY-Greenphire**

### CODE REVIEW

File: RKWorkspace.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>IDEDidComputeMac32BitWarning</key>
```
Comment (erubio-gp) [inline RKWorkspace.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: Whats the purpose of adding this? are we using it somewhere? if not lets take it out @HarshalY-Greenphire

**Review by erubio-gp — CHANGES_REQUESTED** [review]

File: RKWorkspace.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (hunk 1 above)
Comment (HarshalY-Greenphire) [reply RKWorkspace.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: This file is added by xcode it self when i opened the project. It is happening from Xcode 9.3

When i searched, found following reason. 

'Xcode 9.3 adds a new IDEWorkspaceChecks.plist file to a workspace's shared data, to store the state of necessary workspace checks. Committing this file to source control will prevent unnecessary rerunning of those checks for each user opening the workspace. (37293167)'


Reference Link
https://developer.apple.com/library/archive/releasenotes/DeveloperTools/RN-Xcode/Chapters/Introduction.html

Please check Xcode 9.3 -> General -> Resolved Issues. @erubio-gp

Comment (erubio-gp) [reply RKWorkspace.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: Discard the change and see if you can build successfully. I don't think we need this. @HarshalY-Greenphire

Comment (HarshalY-Greenphire) [reply RKWorkspace.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: Sure.

**Pull request merged**